vgcn.yml

```
@@ -2,6 +2,7 @@
 - hosts: my_hosts
```

```suggestion
- hosts: dnb01.bi.privat
```
we build the tar locally and then copy it over to this host